lib/crawlers/rewards.js

```diff
@@ -51,7 +51,9 @@ module.exports = {
       // Fetch last reward events from PolkaScan
       const response = await axios.default.get(`https://api-01.polkascan.io/kusama/api/v1/event?&filter[module_id]=staking&filter[event_id]=Reward`);
 
-      const rewardEvents = response.data.data;
+      const rewardEvents = response.data;
```

Comment:
This is actually wrong. I just added a commit to fix it

Reply:
I'm testing why rewards are not inserted in db ;-)

Reply:
For me it is successfully storing in the local DB:

![image](https://user-images.githubusercontent.com/40721795/76173245-17c90b00-619e-11ea-9d4a-41c973f3d705.png)